test/night_reader_test.rb

```diff
@@ -8,9 +8,17 @@
 
 class NightReaderTest < Minitest::Test
 
-  def test_it_exists
+  def test_it_exists_with_attributes
+    ARGV.replace ["braille.txt", "original_message.txt"]
     night_reader = NightReader.new
     assert_instance_of NightReader, night_reader
+    assert_instance_of FileReader, night_reader.reader
+  end
+
+  def test_it_can_get_braille_to_english_character_character_count
+    ARGV.replace ["braille.txt", "original_message.txt"]
+    night_reader = NightReader.new
+    assert_equal 256, (night_reader.reader.read.size / 6) - (night_reader.reader.read.size / 6 / 80)
```

Comment:
I _think_ this is right. Since there are six characters of braille for every one character of text, the size / 6 should have been 256 but it was 259. 

I believe it's because it's counting the line breaks for every 80 characters as a character. So since there were three line breaks due to the character count, I think that's how 256 became 259. Did math to adjust.